src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -189,9 +223,10 @@ public void initSendable(SendableBuilder sendable) {
     sendable.addBooleanProperty("Climb mode", this::getClimbMode, this::setClimbMode);
     sendable.addDoubleProperty("Left lift encoder", leftClimbLiftEncoder::getIntegratedSensorPosition, null);
     sendable.addDoubleProperty("Right lift encoder", rightClimbLiftEncoder::getIntegratedSensorPosition, null);
-    sendable.addDoubleProperty("Rotate encoder", () -> {
-      return encoderTicksToDegrees(leadClimbRotate.getSelectedSensorPosition());
-    }, null);
+    sendable.addDoubleProperty("Raw rotate encoder", leadClimbRotate::getSelectedSensorPosition, null);
+    sendable.addDoubleProperty("Rotate encoder", () -> encoderTicksToDegrees(leadClimbRotate.getSelectedSensorPosition()), null);
+    sendable.addDoubleProperty("Position Var", () -> position, null);
```

Comment:
when completed values from testing should be removed and any values that are staying should be adjusted to make sure they have correct units